### PR TITLE
Add VC Credential Schema

### PR DESCRIPTION
**Pull request by KendallWeihe**

### CODE REVIEW

File: bindings/web5_uniffi/src/web5.udl [hunk 1]
```diff
@@ -257,10 +257,23 @@ dictionary VerifiableCredentialCreateOptionsData {
   sequence<string>? type;
   timestamp? issuance_date;
   timestamp? expiration_date;
+  CredentialSchemaData? credential_schema;
   string? json_serialized_evidence;
 };
 
 interface VerifiableCredential {
+  [Throws=Web5Error]
+  constructor(
```
Comment (diehuxx) [inline bindings/web5_uniffi/src/web5.udl:266]: what's this second constructor for?

File: crates/web5/src/credentials/credential_schema.rs [hunk 2]
```diff
@@ -0,0 +1,72 @@
+use super::verifiable_credential_1_1::VerifiableCredential;
+use crate::errors::{Result, Web5Error};
+use jsonschema::{Draft, JSONSchema};
+use reqwest::blocking::get;
```
Comment (diehuxx) [inline crates/web5/src/credentials/credential_schema.rs:4]: another spot for us to think about making async in the future

File: crates/web5/src/credentials/mod.rs [hunk 3]
```diff
@@ -1,4 +1,5 @@
 mod create;
+pub mod credential_schema;
```
Comment (diehuxx) [inline crates/web5/src/credentials/mod.rs:2]: `use credentials::credential_schema::CredentialSchema` vs `use credentials::CredentialSchema`. up to you, i prefer the latter.

**Review by diehuxx — APPROVED** [review]

File: bindings/web5_uniffi/src/web5.udl (hunk 1 above)
Comment (KendallWeihe) [reply bindings/web5_uniffi/src/web5.udl]: good callout, this is unnecessary actually. thanks!

File: crates/web5/src/credentials/mod.rs (hunk 3 above)
Comment (KendallWeihe) [reply crates/web5/src/credentials/mod.rs]: nice yeah totally agree

**Review by nitro-neal — APPROVED** [review]

**Pull request merged**